tests/download-upload-package.js

```diff
@@ -0,0 +1,26 @@
+const test = require('ava');
+
+const withPageAndStaticServer = require('./utils/_withPageAndStaticServer');
+
+test('download package programmatically', withPageAndStaticServer, async (t, page) => {
+  const TEST_PACKAGE_HASH = 'QmesnBdGf4yqtC3DH5d1W3GW4qYCtJr6TVfRGBAwQmyCq3'; //avatar.wbn
+  const response = await page.evaluate(downloadFunction, TEST_PACKAGE_HASH);
+  t.true(response);
+});
+
+test('upload package programmatically', withPageAndStaticServer, async (t, page) => {
+  const response = await page.evaluate(uploadFunction, `${t.context.staticUrl}/assets/avatar.wbn`);
```

Comment:
As a minor optimization, can we use a small package here? Each byte costs bandwidth and time for test runs.

Reply:
sure thing!